src/elements/content-sidebar/SidebarResponsiveness.scss

```diff
@@ -0,0 +1,154 @@
+@import '../common/variables';
+
+.u-md-bcs-SidebarNavSign-icon {
+    display: none;
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .bcs-SidebarNavSign-icon {
+            display: none;
+        }
+
+        .u-md-bcs-SidebarNavSign-icon {
+            display: inline;
+            width: 20px;
+            height: 20px;
+            margin-top: 2px; //for visual alignment
+        }
+
+        .bcpr {
+            .bcpr-body {
+                position: static;
+                flex-direction: column;
+            }
+
+            .bcpr-container {
+                position: static;
+            }
+        }
+
+        .preview-body {
+            height: 100%;
+        }
+
+        .be {
+            height: 100%;
+
+            .bcs-activity-feed {
+                width: 100%;
+            }
+        }
+
+        .bcs-activity-feed-comment-input {
+            width: 100%;
+        }
+
+        .be.bcs {
+            flex-direction: column;
+            width: 100%;
+            min-width: unset;
+            max-width: unset;
+            height: 100%;
+            max-height: 48px;
+            -webkit-transition: max-height .5s ease-in-out 0s;
+            transition: max-height .5s ease-in-out 0s;
+        }
+
+        .bcs-content {
+            width: unset;
+            height: 100%;
+
+            .bcs-scroll-content {
+                width: unset;
+            }
+        }
+
+        .be.bcs.bcs-is-open {
+            position: absolute;
+            bottom: 0;
+            z-index: 2;
```

Comment:
Is this needed? If so, does it work properly with all other floating/tethered elements (annotations, tooltips, modals, etc.)?

Reply:
The `z-index` is needed so the drawer displays over the Preview Header when expanded.

![Screen Shot 2022-06-29 at 3 12 30 PM](https://user-images.githubusercontent.com/6400298/176554989-8ba3ed88-b35a-49dd-b8e7-9f8b0887ce1e.png)